ocs_ci/deployment/ibmcloud.py

```diff
@@ -472,17 +474,20 @@ def configure_cloud_credential_operator():
         the cluster while in manual mode.
 
         """
-        # retrieve ccoctl binary from https://mirror.openshift.com
-        version = config.DEPLOYMENT.get("ccoctl_version")
-        source = f"https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/ccoctl-linux.tar.gz"
         bin_dir = config.RUN["bin_dir"]
-        tarball = os.path.join(bin_dir, "ccoctl-linux.tar.gz")
-        logger.info("Downloading ccoctl tarball from %s", source)
-        download_file(source, tarball)
-        cmd = f"tar -xzC {bin_dir} -f {tarball} ccoctl"
-        logger.info("Extracting ccoctl binary from %s", tarball)
-        exec_cmd(cmd)
-        delete_file(tarball)
+        ccoctl_path = os.path.join(bin_dir, "ccoctl")
+        if not os.path.isfile(ccoctl_path):
+            # retrieve ccoctl binary from https://mirror.openshift.com
+            version = config.DEPLOYMENT.get("ccoctl_version")
+            source = f"https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/ccoctl-linux.tar.gz"
+            bin_dir = config.RUN["bin_dir"]
+            tarball = os.path.join(bin_dir, "ccoctl-linux.tar.gz")
+            logger.info("Downloading ccoctl tarball from %s", source)
```

Comment:
can we use f-string format

same in line 488

Reply:
I've just indented the code under the condition.

BTW this is still recommended way of logger formatting in our [guidelines](https://ocs-ci.readthedocs.io/en/latest/coding_guidelines.html#:~:text=logger.info(%22My%20message%20%25s%22%2C%20variable)%20%20%23%20No%20usage%20of%20%25%20after%20string!) ;)


So I am not going to change it.